spec/acceptance/nodesets/amazonlinux-201703-docker.yml

```diff
@@ -0,0 +1,16 @@
+HOSTS:
+  amazonlinux-201703:
+    roles:
+      - agent
+    platform: el-6-x86_64
+    hypervisor: docker
+    image: amazonlinux:2017.03
+    docker_preserve_image: true
+    docker_cmd:
+      - '/sbin/init'
+    docker_image_commands:
+      - 'yum install -y tar wget cronie git upstart'
```

Comment:
How are these dependencies determined?

Reply:
They are things you may need to bootstrap test environment.  I always do git incase git cloning modules for testing.  For something like `upstart` is needed to support `docker_cmd` of `/sbin/init` which is necessary to have functioning container for tests.  Others may be unnecessary but always added as sometimes useful.  `cronie` likely not needed.

Reply:
Further investigation, I believe cronie may have been added to support potentially adding cron jobs via Puppet in other modules and being able to test cron with serverspec: http://serverspec.org/resource_types.html#cron.  Not 100% certain if serverspec uses crontab or looks in /var/spool/cron type situation.